giraffe/src/types/index.ts

```diff
@@ -1,4 +1,3 @@
-import CSS from 'csstype'
 import {CSSProperties, ReactNode} from 'react'
```

Comment:
For our purposes, there is virtually no difference between `CSS.Properties` from the  'csstype' package and `CSSProperties` from 'react'. Since there is no difference, for consistency, it is better to use `CSSProperties` everywhere rather than having isolated usages of `CSS.Properties` for no apparent reason.